### DISCUSSION TITLE
Problem sending emails

### DISCUSSION THREAD

**Discussion by Toni202012569** [postalserver/postal#2659] [Help with using Postal]
Messages are queued. However, if in our personal account we click the Redeliver message button, the letter is sent. There are no errors in the logs. With background mode, emails are not sent automatically.

**Comment by willpower232** [comment]
What is the output of `postal status`?

**Reply by Toni202012569** [reply]
Good evening, where can I see this status?

**Reply by Toni202012569** [reply]
And I have one more question, why at maximum load is the throughput only 200 letters per minute? How can this figure be increased?
I also noticed that for some reason the server slows down the speed of sending letters, although there are quite a lot of letters in the queue to be sent.

**Reply by willpower232** [reply]
if you have a regular installation and run `postal status` from the command line then you should see whether or not everything is running

if you would like to process emails faster then you should have a look at this little list here
https://github.com/postalserver/postal/discussions/1461#discussioncomment-1119403
